packages/connect/package.json

```diff
@@ -91,6 +91,7 @@
     "devDependencies": {
         "@trezor/trezor-user-env-link": "workspace:*",
         "@types/karma": "^6.3.8",
+        "@types/w3c-web-usb": "^1.0.10",
```

Comment:
Shows up as an unused dependency, which makes sense since it's used in `@packages/transport` not `@packages/connect`

Reply:
and it shouldn't be needed since it is listed in `usb` package 

<img width="385" alt="image" src="https://github.com/user-attachments/assets/f8653e16-e1b0-45b5-9f90-f6bc5a8bf20c">